onnx/defs/schema.h

```diff
@@ -1070,14 +1070,12 @@ class OpSchema final {
     return opset_version_to_function_builder_.find(opset_version) != opset_version_to_function_builder_.end();
   }
 
-  OpSchema& SetContextDependentFunctionBodyBuilder(
-      ContextDependentFunctionBodyBuilder,
-      int opset_version = kUninitializedSinceVersion);
+  OpSchema& SetContextDependentFunctionBodyBuilder(ContextDependentFunctionBodyBuilder, int opset_version);
 
   bool BuildContextDependentFunction(
       const FunctionBodyBuildContext& ctx,
       FunctionProto& function_proto,
-      int requested_opset_version = OpSchema::kUninitializedSinceVersion) const;
```

Comment:
Wondering: is `OpSchema::kUninitializedSinceVersion` used anywhere else, or can it be removed?